pkg/build/controller/config_controller.go

```diff
@@ -28,8 +28,12 @@ func (e ConfigControllerFatalError) Error() string {
 
 // IsFatal returns true if err is a fatal error
 func IsFatal(err error) bool {
-	_, isFatal := err.(ConfigControllerFatalError)
-	return isFatal
+	switch err.(type) {
+	case ConfigControllerFatalError, *ConfigControllerFatalError:
```

Comment:
I considered it, but nothing is going to stop someone else from going it in the future, so i'd want to check both regardless.